swap_meet/vendor.py

```diff
@@ -1,2 +1,51 @@
 class Vendor:
-    pass
+    def __init__(self, inventory=[]):
+        self.inventory = inventory
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+
+        self.inventory.remove(item)
```

Comment:
Is there a way we could use `remove` without needing the guard clause (or any other `if`)? How does `remove` report not finding an item? Can we handle that?